src/service_config/task_queue_config.py

```diff
@@ -52,6 +53,9 @@ def configure_task_queue(service,
         smtp["user"] = "montagu"
         smtp["password"] = get_secret("email/password")
 
+    if is_prod:
+        config["servers"]["youtrack"]["token"] = \
+            get_secret("vimc-robot/youtrack-task-queue-token")
```

Comment:
In the absence of a provided config value (e.g. in the test context on BuildKite) the task q will look for an environment variable called "YOUTRACK_TOKEN" (see https://github.com/vimc/montagu-task-queue/pull/24/files#diff-f81481734ef4384fb31e9f3cb8d3afacf27470dedc4210533a3fc4747419c33dR34) which now exists on the BuildKite agents.